packages/interactivity-router/src/index.ts

```diff
@@ -314,28 +349,32 @@ export const { state, actions } = store( 'core/router', {
 		 * @param {boolean} [options.force] Force fetching the URL again.
 		 * @param {string}  [options.html]  HTML string to be used instead of fetching the requested URL.
 		 */
-		prefetch( url, options = {} ) {
+		prefetch( url: string, options: PrefetchOptions = {} ) {
 			const { clientNavigationDisabled } = getConfig();
 			if ( clientNavigationDisabled ) {
 				return;
 			}
 
 			const pagePath = getPagePath( url );
 			if ( options.force || ! pages.has( pagePath ) ) {
-				pages.set( pagePath, fetchPage( pagePath, options ) );
+				pages.set(
+					pagePath,
+					fetchPage( pagePath, { html: options.html } )
```

Comment:
fetchPage signature is: 

```ts
const fetchPage = async ( url: string, { html }: { html: string } )
```

TS is not happy if we pass the whole `options` object which contains additional properties.